package.json

```diff
@@ -51,8 +51,11 @@
     "karma-webpack": "^1.7.0",
     "legacy-loader": "0.0.2",
     "less": "^2.3.1",
+    "node-sass": "^3.4.2",
```

Comment:
`npm i --save-dev sass-loader` output a warning requiring a peer dependency of node-sass at 3.4.2, so I obeyed npm's output and added it. Not that I fully understand the nuance of peer dependencies.